admin/js/boldgrid-backup-admin-settings-autoupdate.js

```diff
@@ -78,6 +78,17 @@ BOLDGRID.SETTINGS = BOLDGRID.SETTINGS || {};
 					$( '#timely-updates-days-hidden' ).prop( 'disabled', true );
 				}
 			} );
+
+			$( '#timely-updates-days' ).on( 'change', self._validateDays );
+		},
+
+		_validateDays: function( event ) {
```

Comment:
Maybe we can just relay on the (1) [html form validation](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/number) and (2) a check in php, rather than a (3rd) additional option of checking with javascript?

Reply:
I removed the js validation and added php validation for this instead.